website/docs/components/code-block/partials/code/component-api.md

```diff
@@ -0,0 +1,41 @@
+## Component API
+
+This component uses [prism.js](https://prismjs.com/) under the hood.
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="value" @type="string">
+    The text/code content for the CodeBlock.
```

Comment:
```suggestion
    The text/code content for the `CodeBlock`.
```

Reply:
Addressed in [4ea291b](https://github.com/hashicorp/design-system/pull/1749/commits/4ea291bbca9846b2dd2d1bbe4e6151c6fca2e066)